README.md

```diff
@@ -156,6 +156,7 @@ only landed in the `v4.x` branch when a new `v4.x` release is being prepared.**
 [Boron]: https://nodejs.org/download/release/latest-boron/
 [Carbon]: https://nodejs.org/download/release/latest-carbon/
 [Dubnium]: https://nodejs.org/download/release/latest-dubnium/
+[Erbium]: https://nodejs.org/download/release/latest-erbium/
```

Comment:
We'll need to remember to add `erbium` to https://github.com/nodejs/nodejs-latest-linker/blob/46a6c7069a2c30028d149985327a34a71e69878e/latest-linker.js#L10-L15 for this link to get created. I'm unsure of the sequencing (whether it should happen before or after the release).

Reply:
Thanks, I opened the PR: https://github.com/nodejs/nodejs-latest-linker/pull/3